services/storage/tests/test_dsm.py

```diff
@@ -177,16 +179,20 @@ def _create_file_meta_for_s3(postgres_url, s3_client, tmp_file):
     assert len(data) == 2
 
 #NOTE: Below tests directly access the datcore platform, use with care!
-@pytest.mark.travis
 def test_datcore_fixture(datcore_testbucket):
+    if not has_datcore_tokens():
```

Comment:
what is the meaning of that test? does that disable it on travis?

Reply:
> what is the meaning of that test? does that disable it on travis?

It labels the test, such that it can be skip upon request via, e.g. command line